frontend/styles/globals.scss

```diff
@@ -0,0 +1,128 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;500;600&display=swap');
+
+@layer base {
+  @font-face {
+    font-family: 'Typo Round';
+    src: url('/fonts/Typo_Round_Regular_Demo.otf') format('opentype');
```

Comment:
We really shouldn't use proprietary fonts here, I've asked Arhella to replace it with a free to use alternative

Reply:
Yeah I had this question because this was a demo font but it was just to add one in the meantime :)